fairscale/nn/pipe/async_pipeline.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+import logging
+from threading import Event
+from typing import List, Optional
+
+import torch
+
+from .async_schedule import AsyncEventLoop
+from .microbatch import Batch
+from .multiprocess_pipeline import MultiProcessPipeline
+from .skip.tracker import SkipTrackerThroughPotals
+
+
+class AsyncPipeline(MultiProcessPipeline):
+    def run(self, training: bool, batches: List[Batch], event: Optional[Event]) -> None:
+
+        """Runs pipeline parallelism.
+
+        It modifies the given batches in place.
+
+        """
```

Comment:
nit: Can we update this comment to make this more descriptive of what AsyncPipeline does?